tagmedia

```diff
@@ -59,6 +59,13 @@ use constant {
   MBR_LENGTH => 0x200,
 };
 
+# Some SUSE specific constants.
+#
+use constant {
+  # signature block starts with this string
+  SIGNATURE_MAGIC => "7984fc91-a43f-4e45-bf27-6d3aa08b24cf"
```

Comment:
Q: Where this comes from? Just in case we need to update that...

Reply:
added links to code